### DISCUSSION TITLE
Macros not working anymore in Goland

### DISCUSSION THREAD

**Discussion by mmelitzer** [JetBrains/ideavim#949] [Q&A] [answered]
I've no idea what happened but suddenly macro execution stopped working. No change in config that I'm aware of but possible that something was updated (Goland, OS, ideavim, ...). I also use PyCharm and there it works flawlessly, generally have the same plugins installed in both and nothing added lately. I can see the `@` in the bottom of the IDE when trying to apply a macro (also can print out the macro via `:echo @<ch>`) but when pressing the next character it just gets ignored (only `@` remains at bottom), pressing yet another char behaves as it's the first char pressed (e.g. i -> insert) or added (e.g. g -> @g).
Checked Keymap and `@` is not assigned to anything, also no special `@` assignment in the OS (using OS X).

Anything I can check to see what's going on or what could be the issue.

**Comment by lippfi** [comment, marked as answer]
Could you please tell your IDE and IdeaVim versions?

**Reply by mmelitzer** [reply]
GoLand 2023.3.5 Build #GO-233.14808.20
IdeaVim 2.12.0

**Reply by lippfi** [reply]
It is a knows issue and it is fixed in future IdeaVim versions

**Reply by mmelitzer** [reply]
K, thanks, was driving me nuts
